Bitrise Qt5 build bot is failing

This is probably caused by a wrong selection of the clang compiler version:

These errors started to pop up on Bitrise and also on my macOS:
```
In file included from ../../../include/mbgl/util/constants.hpp:3:
../../../include/mbgl/util/chrono.hpp:36:49: error: no template named 'enable_if_t' in namespace 'std'; did you mean 'enable_if'?
template <class Rep, class Period, class = std::enable_if_t<
```
```
../../../include/mbgl/util/optional.hpp:8:37: error: no type named 'optional' in namespace 'std::experimental'
using optional = std::experimental::optional<T>;
                 ~~~~~~~~~~~~~~~~~~~^
```
I'm going to disable Qt5 build bot until we find a fix for this.

/cc @tmpsantos @kkaefer

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.1)
 project(mbgl LANGUAGES CXX C)
+set(CMAKE_CXX_STANDARD 14)
 
 include(cmake/mbgl.cmake)
 
@@ -42,6 +43,11 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra -Wshadow -Werro
 if(APPLE)
     # -Wno-error=unused-command-line-argument is required due to https://llvm.org/bugs/show_bug.cgi?id=7798
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error=unused-command-line-argument")
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX)
+    # https://svn.boost.org/trac/boost/ticket/9240
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fext-numeric-literals")
 endif()
 
 if(NOT EXISTS ${CMAKE_SOURCE_DIR}/platform/${MBGL_PLATFORM}/config.cmake)

--- a/platform/qt/app/mapwindow.cpp
+++ b/platform/qt/app/mapwindow.cpp
@@ -297,5 +297,15 @@ void MapWindow::resizeGL(int w, int h)
 void MapWindow::paintGL()
 {
     m_frameDraws++;
+
+#if QT_VERSION < 0x050400 && defined(__APPLE__)
+    // XXX GL framebuffer is valid only after first attempt of painting on
+    // older versions of Qt on macOS.
+    // See https://bugreports.qt.io/browse/QTBUG-36802 for details.
+    static bool first = true;
+    if (!first) m_map.render();
+    first = false;
+#else
     m_map.render();
+#endif
 }

